Document Lottie issues (#994)
cashapp/paparazzi: README.md

android.jetifier.ignorelist=android-base-common,common
```

Lottie
--------

> When taking screenshots of Lottie animations, Paparazzi can throw weird exceptions ([#494](https://github.com/cashapp/paparazzi/issues/494), [#630](https://github.com/cashapp/paparazzi/issues/630)). 
> 
> As a workaround, you need to force Lottie to not run on the background thread:

Reviewer: ```suggestion
When taking screenshots of Lottie animations, you need to force Lottie to not run on a background thread, otherwise Paparazzi can throw exceptions [#494](https://github.com/cashapp/paparazzi/issues/494), [#630](https://github.com/cashapp/paparazzi/issues/630). 
```
